examples/cpp/sync/authentication.cpp

```diff
@@ -142,39 +142,34 @@ void testSignInWithAppleAuthSyntax() {
   // :snippet-end:
 }
 
-// TODO: This has been renamed to realm::App::credentials::google_auth_code
-// but is not working here. Figure out how to use it in the updated SDK.
-// void testSignInWithGoogleAuthCodeSyntax() {
-//   auto myAuthCode = "some auth code string";
-//   auto authCode = realm::App::credentials::auth_code{myAuthCode};
-
-//   // :snippet-start: google-auth-code
-//   auto appConfig = realm::App::configuration();
-//   appConfig.app_id = APP_ID;
-//   auto app = realm::App(appConfig);
-
-//   // The auth_code below is the user's server auth code you got from Google
-//   auto user = app.login(realm::App::credentials::google(authCode)).get();
-//   // :snippet-end:
-// }
+void testSignInWithGoogleAuthCodeSyntax() {
+  auto myAuthCode = "some auth code string";
```

Comment:
```suggestion
  // The user's server auth code you got from Google
  auto myAuthCode = "auth_code_from_google"; 
```
maybe include this in the snippet?